presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java

```diff
@@ -295,17 +292,15 @@ private static URI getQueuedUri(QueryId queryId, Slug slug, long token, UriInfo
     }
 
     /**
-     * Instantiates an object of type {@link QueryResults} with information about
-     * all query related metadata.
-     *
-     * @param queryId an object containing the query identifier.
-     * @param nextUri an object that checks if the query is completed or directs another uri.
-     * @param queryError an object responsible for reporting error in the query.
-     * @param uriInfo an object responsible for taking metadata from the uri.
-     * @param elapsedTime time elapsed until the query response.
-     * @param queuedTime queue waiting time.
+     * Instantiates an object that contains all information about query metadata.
      *
-     * @return a {@link QueryResults} object.
+     * @param queryId an object containing the query identifier
+     * @param nextUri an object that contains the URI where information about query can be retrieved
+     * @param queryError an object responsible for reporting the error that happened during the query's execution
+     * @param uriInfo an object that will be used to construct the query representation in web interface
+     * @param elapsedTime the time elapsed until the query obtains its response
+     * @param queuedTime the time the query waited to be executed
+     * @return an object with all query's metadata
```

Comment:
why not keep the link to {@link QueryResults} in the return statement?

Reply:
https://softwareengineering.stackexchange.com/questions/199293/is-repeating-links-to-the-same-class-in-a-single-javadoc-comment-a-bad-practice